pandas/plotting/_matplotlib/timeseries.py

```diff
@@ -9,6 +10,7 @@
 )
 
 import numpy as np
+import pandas
```

Comment:
if you must, do `import pandas as pd`.  but we try to avoid that in favor or more specific imports where possible